### PR TITLE
Doc improvements

### PR DESCRIPTION
**Pull request by kit-ty-kate**
The change of the advised way to call `install.sh` was proposed in https://github.com/ocaml/platform-blog/pull/77 and includes:
* `<()` is a bash syntax, so people using this command in `sh` or `csh` will see the command fail
* `curl` without `-fS` will output things in stdout when something goes wrong (404, and other network failure) and can cause random shell commands being executed. This is what https://brew.sh/ recommend for instance.

### CODE REVIEW

**Comment by rjbou** [comment]
+ update of readme badges, release instructions, changelog for api follow-up

File: master_changes.md [hunk 1]
```diff
@@ -120,7 +124,15 @@ Prefixes used to help generate release notes, changes, and blog posts:
   * fish: fix deprecated redirection syntax `^` [#4736 @vzaliva]
 
 ## Doc
-  *
+  * Standardise `macOs` use [#4782 @kit-ty-kate]
```
Comment (dra27) [inline master_changes.md:127]:
```suggestion
  * Standardise `macOS` use [#4782 @kit-ty-kate]
```
🙂

File: master_changes.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ Prefixes used to help generate release notes, changes, and blog posts:
 * ◈ New option/command/subcommand
 * [BUG] for bug fixes
 * [NEW] for new features (not a command itself)
+* 🕮i api update
```
Comment (dra27) [inline master_changes.md:8]: What's the emoji bit for, or is my browser being strange?

I'm not utterly convinced that this is a good idea, but let's see how it goes!

**Review by dra27 — COMMENTED** [review]
One, or possibly two, typos but otherwise LGTM!

File: master_changes.md (hunk 2 above)
Comment (rjbou) [reply master_changes.md:8]: héhé, it is just a book for _library_ , maybe we should have the more intelligible `[API]`. The idea is to point API breakage/updates that can affect library users

**Pull request merged**